src/test/e2e/api/admin/favorites.e2e.test.ts

```diff
@@ -162,3 +175,47 @@ test('should be able to unfavorite feature', async () => {
         favorite: false,
     });
 });
+
+test('should be favorited in projects list', async () => {
+    await favoriteProject();
+
+    const { body } = await app.request
+        .get(`/api/admin/projects`)
+        .set('Content-Type', 'application/json')
+        .expect(200);
+
+    expect(body.projects).toHaveLength(1);
+    expect(body.projects[0]).toMatchObject({
+        name: 'Default',
+        favorite: true,
+    });
+});
+
+test('should be favorited in single project endpoint', async () => {
+    await favoriteProject();
+
+    const { body } = await app.request
```

Comment:
same as above, I'd keep one abstraction level in a test. When we go for helper domain method let's stick to it